catalog/admin/admin-poller-service/src/test/java/org/codice/ddf/catalog/admin/poller/AdminPollerServiceBeanTest.java

```diff
@@ -145,10 +160,57 @@ public void testAllSourceInfo() {
   }
 
```

Comment:
✏️ We should have a test where multiple SourceDescriptors are returned and the RuntimeException is thrown from `Iterables.getOnlyElement`.

Reply:
Done.